eclipsePlugin/test/de/tobject/findbugs/reporter/ThrottledProgressMonitorTest.java

```diff
@@ -0,0 +1,52 @@
+package de.tobject.findbugs.reporter;
```

Comment:
Test code and test dependencies do not belong to the production plugin project. There is nothing to discuss, this is how Eclipse plugins are structured and deployed. We have had many Eclipse tests running before, now thanks to all gradle changes we don't have it, not even manual one. Not even test code is compiled anymore - because if it would compile, one would see that we managed to break the test code too.

If you wish to add test code (which is great), please do this into the test plugin and please help to fix the gradle build to make the test plugin working again.

Until then I would just don't touch Eclipse plugin code, because no one knows which regressions one can introduce.